src/docs/framework-integration/react.md

```diff
@@ -202,8 +218,153 @@ import { MyComponent } from 'component-library-react';
 
 With that, your component is now available to be used like any other React component.
 
+## API
+
+### componentCorePackage
+
+**Optional**
+
+**Default: The `components.d.ts` file in the Stencil project's `package.json` types field**
+
+**Type: `string`**
+
+The title of the Stencil package where components are available for consumers (i.e. the `name` property value in your Stencil project's `package.json`).
```

Comment:
Optional word smithing to use some of the same nomenclature as the rest of the doc:
```suggestion
The name of the Stencil package where components are available for consumers (i.e. the value of the `name` property in your Stencil component library's `package.json`).
```

Reply:
@rwaskiewicz I like this change, but this same thing probably needs to happen in the Angular and Vue docs as well. So, I'll hold off on making this swap until we merge all these back into one PR so then I can just change them all at once